django/utils/decorators.py

```diff
@@ -69,8 +79,9 @@ def dummy(*args, **kwargs):
             return obj
 
         return _wrapper
-
-    update_wrapper(_dec, decorator, assigned=available_attrs(decorator))
+    # we really don't care to make _dec look similar to an iterable, it is rather meaningless
```

Comment:
I had skipped previously this way, in case decorator has ``__iter__``, we just skip ``update_wrapper()`` on ``_dec()`` it since it as became meaningless.